backend/apps/sandbox/serializers.py

```
@@ -158,4 +158,13 @@ class Meta:
             "files",
             "created_at",
         ]
+        read_only_fields = ["id", "created_at"]
+
+
+from .models import CollabSession, CollabSessionLog
+
+class CollabSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CollabSession
+        fields = ["id", "project", "allowed_users", "created_at", "is_active"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)